src/sync-endpoint-cloud-setup.rst

```diff
@@ -74,60 +74,93 @@ Setting up a Droplet
 
-5. After the authentication is set up, you can choose to name the droplet; then scroll down and click the :guilabel:`Create Droplet` button. This might take a few minutes to set up.
+  .. image:: /img/setup-digital-ocean/click-create-droplet.png
+    :width: 600
 
 .. _sync-endpoint-setup-digital-ocean-dns:
 
 Setting up a DNS Record
 """"""""""""""""""""""""
 
-1. On the resources tab of the main DigitalOcean page, click on the :guilabel:`Droplet` you created.
+1. On the resources tab of the main DigitalOcean page, hover over the IP address of the droplet you just created and click on the :guilabel:`Copy` text that shows up to copy the IP address of the droplet.
 
   .. image:: /img/setup-digital-ocean/do-droplets.png
    :width: 600
 
-2. Obtain the IP address of the droplet you created.
+2. Log into your account for your domain name registrar and DNS provider. See :ref:`Acquiring a domain name<sync-endpoint-setup-domain>` for more information and a list of registrars and DNS providers.
 
-3. Log into your account for your domain name registrar and DNS provider. See :ref:`Acquiring a domain name<sync-endpoint-setup-domain>` for more information and a list of registrars and DNS providers.
+3. Add a dns 'A' record for the domain or subdomain you would like to use for the Sync Endpoint with your droplet's IP address.
 
-4. Add a dns 'A' record for the domain or subdomain you would like to use for the Sync Endpoint with your droplet's IP address.
-
-.. _sync-endpoint-setup-digital-ocean-connecting:
+.. _sync-endpoint-setup-digital-ocean-firewall:
 
-Connecting to your Droplet
-"""""""""""""""""""""""""""""
+Enabling a firewall to prevent unintended traffic
+"""""""""""""""""""""""""""""""""""""""""""""""""""
 
 1. On the resources tab of the main DigitalOcean page, click on the :guilabel:`Droplet` you created.
 
-  .. image:: /img/setup-digital-ocean/do-droplets.png
+  .. image:: /img/setup-digital-ocean/syncendpoint-droplet.png
+   :width: 600
+
+2. On the droplet page, click on :guilabel:`Networking`, this will take you to the droplet's networking page.
+  .. image:: /img/setup-digital-ocean/do-networking.png
    :width: 600
 
-2. Now, click on the :guilabel:`Console` link in the upper-right corner of the page
+3. Next, scroll down to the :guilabel:`Firewalls` section and click on the :guilabel:`Edit` button.
+
+  .. image:: /img/setup-digital-ocean/do-edit-firewall.png
+   :width: 600
+
+4. On the :guilabel:`Firewalls` tab of the :guilabel:`Networking` page,  and click on the :guilabel:`Create Firewall` button.
+
+  .. image:: /img/setup-digital-ocean/do-firewall.png
+   :width: 600
+
+5. Set a name for your firewall and modify the inbound rules to match the inbound rules specified in the picture below (SSH, HTTP, HTTPS and port (40000) for admin interface). Attach the firewall to the desired droplet. Leave the outbound rules as-is.
```

Comment:
Need to add a warning do not leave 40000 open. We should think about adding SSH tunnel instructions.

Reply:
Do you mean opening the port to a specific IP address instead of leaving it open to traffic from anywhere?